libs/designer-ui/src/lib/mixedinputeditor.tsx

```diff
@@ -0,0 +1,110 @@
+import type React from 'react';
```

Comment:
The LAUX team may ask you to write some unit tests for this component, FYI

Reply:
Updated with tests now